CONTRIBUTING.md

```
@@ -55,4 +56,4 @@ Set this before cloning the repository.
 ## License
 
 QtPass is released under the GNU GPL v3.0 license.
-<https://www.gnu.org/licenses/gpl-3.0.html>
+<https://www.gnu.org/licenses/gpl-3.0.html>
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)